README.md

```diff
@@ -1,2 +1,162 @@
-# choreo-samples
-This contains integration and service samples displayed in choreo.
+# Choreo Samples
+
+This repository contains a collection of sample projects ready to deploy in [Choreo](https://console.choreo.dev/), a developer platform for easily taking your work to production. These samples demonstrate various use cases and provide examples for different scenarios.
+
+## Samples
+
+This repository contains the following samples:
+
+- [azure-cosmosdb-data-as-rest-api](/azure-cosmosdb-data-as-rest-api)
+- [byoc-time-logger-schedule](/byoc-time-logger-schedule)
+- [capsulecrm-opportunities-lost-to-slack-channel-message](/capsulecrm-opportunities-lost-to-slack-channel-message)
+- [capsulecrm-opportunities-won-to-gmail-email](/capsulecrm-opportunities-won-to-gmail-email)
+- [circuit-breaker](/circuit-breaker)
+- [content-based-routing](/content-based-routing)
+- [covid19-statistics-to-email](/covid19-statistics-to-email)
+- [csv-processing](/csv-processing)
```

Comment:
Having a list of samples in the readme will be hard to maintain. Shall we get rid of the list? If we including a list it is better to organise it with some category.

Reply:
Removed the list of samples for the time being ceecf294fef50f0d2962836f5320343c73addf32